### PR TITLE
added steps for creating a key and additional installation notes

### PR DESCRIPTION
**Pull request by lauraweindorf**
I was attempting to add the steps to create a key (for the registration part) and some other notes, and ended up breaking things up a bit to integrate that info. I liked the simplicity of the instructions before, but I'm hoping this makes it more bullet-proof for people who aren't familiar with running a full-node.

### CODE REVIEW

File: docs/validators-and-full-nodes/run-full-node-mainnet.md [hunk 1]
```diff
@@ -65,12 +112,14 @@ secretd configure-secret node-master-cert.der "$SEED"
...
+```
 
+#### Start your node as a service:
+
+```bash
 sudo systemctl enable secret-node
 
 sudo systemctl start secret-node # (Now your new node is live and catching up)
-
-secretcli config node tcp://localhost:26657
```
Comment (toml01) [inline docs/validators-and-full-nodes/run-full-node-mainnet.md:73]: Why did you removed this line here?

**Review by toml01 — COMMENTED** [review]
Looks good, thanks!

File: docs/validators-and-full-nodes/run-full-node-mainnet.md (hunk 1 above)
Comment (lauraweindorf) [reply docs/validators-and-full-nodes/run-full-node-mainnet.md:73]: I moved it down below the `journalctl` and added some more info after (looking at the `status` and what indicates it's fully sync'd).

Comment (toml01) [reply docs/validators-and-full-nodes/run-full-node-mainnet.md:73]: Yeah I saw that, I just thought it made more sense in the original place. Doesn't really matter :)

**Pull request merged**